### PR TITLE
Add html to match wireframes

### PR DESCRIPTION
**Pull request by RaynasDad**

### CODE REVIEW

**Review by axamei — CHANGES_REQUESTED** [review]
please add the images to the `src` attribute of the `img` tag. 

book an appointment if you need assistance.

**Comment by RaynasDad** [comment]
@axamei This really confused me, because I had already added all these images in the img tags.  (I forgot to hit Save.)  Thankfully, it was all still in my code editor.

File: about.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Archibald Family Website</title>
+</head>
+<body>
+   <header>
+    <nav>
+        <img src="" alt="Archibald Family Website logo">
```
Comment (axamei) [inline about.html]: Image missing

File: about.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <img src="" alt="fancy About image">
```
Comment (axamei) [inline about.html]: Image missing

File: about.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+   </header>
+   <main>
+    <h1>About This Site</h1>
+    <p>This is a website designed to help you, our family and friends, keep up with all of us in the Archibald Family</p>
+   </main>
+   <footer>
+    <img src="" alt="Archibald Family footer image">
```
Comment (axamei) [inline about.html]: Image missing

File: index.html [hunk 4]
```diff
@@ -3,9 +3,56 @@
...
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Archibald Family Website</title>
 </head>
 <body>
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/zcIcK2XjeS0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+   <header>
+    <nav>
+        <img src="" alt="Archibald Family Website logo">
```
Comment (axamei) [inline index.html]: Image missing

File: index.html [hunk 5]
```diff
@@ -3,9 +3,56 @@
...
+        <img src="" alt="Archibald Family Photo">
```
Comment (axamei) [inline index.html]: Image missing

File: index.html [hunk 6]
```diff
@@ -3,9 +3,56 @@
...
+            <li><a href="about.html">About</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+   </header>
+   <main>
+    <div>
+        <ul>
+            <li>
+                <img src="" alt="picture of Matt">
```
Comment (axamei) [inline index.html]: Image missing

File: index.html [hunk 7]
```diff
@@ -3,9 +3,56 @@
...
+                <a href="matt.html">Matt</a>
+            </li>
+            <li>
+                <img src="" alt="picture of Jenn">
```
Comment (axamei) [inline index.html]: Image missing

File: index.html [hunk 8]
```diff
@@ -3,9 +3,56 @@
...
+                <a href="jenn.html">Jenn</a>
+            </li>
+        </ul>
+    </div>
+    <div>
+        <ul>
+            <li>
+                <img src="" alt="picture of Julia">
```
Comment (axamei) [inline index.html]: Image missing

File: index.html [hunk 9]
```diff
@@ -3,9 +3,56 @@
...
+                <a href="julia.html">Julia</a>
+            </li>
+            <li>
+                <img src="" alt="picture of Rayna">
```
Comment (axamei) [inline index.html]: Image missing

File: index.html [hunk 10]
```diff
@@ -3,9 +3,56 @@
...
+                <a href="rayna.html">Rayna</a>
+            </li>
+            <li>
+                <img src="" alt="picture of Josiah">
```
Comment (axamei) [inline index.html]: Image missing

File: index.html [hunk 11]
```diff
@@ -3,9 +3,56 @@
...
+                <a href="josiah.html">Josiah</a>
+            </li>
+            <li>
+                <img src="" alt="picture of Skye">
```
Comment (axamei) [inline index.html]: Image missing

File: index.html [hunk 12]
```diff
@@ -3,9 +3,56 @@
...
+                <a href="skye.html">Skye</a>
+            </li>
+        </ul>
+    </div>
+   </main>
+   <footer>
+    <img src="" alt="Archibald Family footer image">
```
Comment (axamei) [inline index.html]: Image missing

File: matt.html [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Archibald Family Website</title>
+</head>
+<body>
+   <header>
+    <nav>
+        <img src="images/family crest.jpg" alt="Archibald Family crest">
```
Comment (axamei) [inline matt.html]: Please rename the image to not include spaces, e.g., `family-crest.jpg`

File: matt.html [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+        <img src="images/may the fork be with you.jpg" alt="picture of Matt with Yoda on his shoulder, holding up a fork that says 'may the fork be with you'">
```
Comment (axamei) [inline matt.html]: Please rename the image to not include spaces

File: matt.html [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+            <li>
+                <img src="images/mainlibrary_woodmere2.png" alt="Traverse Area District Library">
+                <a href="library.html">Library</a>
+            </li>
+        </ul>
+    </div>
+    <div>
+        <ul>
+            <li>
+                <img src="images/singing at Barns.jpg" alt="picture of Matt holding a guitar, singing at a microphone with Rayna and Jenn">
```
Comment (axamei) [inline matt.html]: Please rename the image to not include spaces

**Review by axamei — CHANGES_REQUESTED** [review]
There are a few more pages missing images that you need to save and commit.

**Review by axamei — CHANGES_REQUESTED** [review]
There are a few more pages missing images that you need to save and commit.

**Comment by RaynasDad** [comment]
@axamei Ugh!  Sorry!  I think I have them all now.  I guess I assumed that I was saving the whole project, not just the page I was on.

**Review by axamei — APPROVED** [review]

**Pull request merged**